qfdmo/admin/acteur.py

```diff
@@ -225,6 +264,9 @@ def get_readonly_fields(self, request, obj=None):
         return readonly_fields
 
 
+### Acteur
```

Comment:
https://www.codu.co/articles/using-regions-in-vs-code-to-organize-your-code-womhkang

Est-ce qu’on ne pourrait pas utiliser les ˋ#regionˋ ？